plugins/web/opentelemetry-plugin-user-interaction/src/userInteraction.ts

```diff
@@ -80,6 +79,12 @@ export class UserInteractionPlugin extends BasePlugin<unknown> {
     }
   }
 
+  /**
+   * Controls whether or not to create a span, based on the event type.
+   */
+  protected allowEventType(eventType: string): boolean {
```

Comment:
this should be private as it is not used by any subclass?

Prefix protected and private methods with _

Reply:
I wanted to leave a way for users to specialize their own list of "interesting enough" events (possibly even dynamic, unlike with a config item).  If that's not cool I can make it private.  Either way I'll prefix with `_`.

Reply:
By writing an instrumentation module which extends ours? that is an interesting idea.

Reply:
Also: perhaps the lint rules could be expanded to enforce the naming?